content/faq/bookmarks/how-to-import/index.es.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Cómo puedo importar los marcadores?
+description: ""
+
+taxonomies:
+  faq: ["Bookmarks"]
+
+extra:
+  order: 20
+---
+
+Puede importar marcadores enviados desde Organic Maps o aplicaciones de terceros si exportan marcadores en formato KML, KMZ, GPX:
+
+1. En Android, puede importar marcadores desde carpetas y aplicaciones (incluido Maps.Me). Presione un botón de estrella para abrir marcadores, presione "Importar marcadores" y seleccione la unidad o carpeta con archivos KML, KMZ, GPX.
+
+2. Envía a ti mismo los archivos KML, KMZ, GPX por correo electrónico.* También puede usar mensajería instantánea o almacenamiento en la nube, como iCloud o Google Drive.
```

Comment:
What's that asterisk `*` symbol?

Reply:
I think it's a typo, it should be removed.

Reply:
@strump ^^^